src/main/io/ledstrip.c

```diff
@@ -945,14 +964,22 @@ void updateLedStrip(void)
 
```

Comment:
quite a lot of counters and high code repetition. This may be handled as loop with template